File: angular/src/app/shared/service/product/models/create-update-product-dto.ts
import { EntityDto } from '@abp/ng.core';

export class CreateOrEditProductDto extends EntityDto<string>{
  productName: string;
  productdescription: string;
  categoryId: string;
  price: number;
  quantity: number;
  imgSource: string;
  size:string;

  constructor(initial: Partial<CreateOrEditProductDto> = {}) {
    super(initial);
  }
}

File: aspnet-core/src/Selling_Website.Application.Contracts/Products/Dto/CreateUpdateProductDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace Selling_Website.Products.Dto
{
    public class CreateUpdateProductDto : EntityDto<Guid?>
    {
        public string ProductName { get; set; }
        public string ProductDescription { get; set; }
        public Guid? CategoryId { get; set; }
        public int? Price { get; set; }
        public int? Quantity { get; set; }
        public string ImgSource { get; set; }
    }
}
